### PR TITLE
Fetch references to combine cross repositories pull requests

### PR DESCRIPTION
**Pull request by thibaultduponchelle**
Hello @bobvanderlinden,

Thank you for this GitHub action which is very cool 👍🏼 

I got systematic `not something we can merge` when combining at least one cross-repository pull request hence I added some "fetch magic" to retrieve pull requests references.

I tested on my side.

Would you kindly look at this and merge 😄

### CODE REVIEW

File: gitconfig [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[remote "origin"]
+    fetch = +refs/pull/*/head:refs/remotes/origin/pr/*
```
Comment (bobvanderlinden) [inline gitconfig]: Isn't this fetching all PRs, regardless of their label?
I'm wondering whether it isn't possible to `git fetch origin ${sha}` or even `git fetch ${shas[@]}`?

Comment (thibaultduponchelle) [reply gitconfig]: Good point, I don’t know but I can search

**Comment by thibaultduponchelle** [comment]
Ok, updated with `git fetch origin "${shas[@]}"` and tested. This is much cleaner and simpler like this 😀

**Review by bobvanderlinden — APPROVED** [review]
Nice! Thanks for the change and testing 👍

**Pull request merged**